content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -68,6 +68,9 @@ Follow these instructions to prepare a kind cluster for Istio installation.
     Deleting cluster "istio-testing" ...
     {{< /text >}}
 
+## Setup MetalLB for kind
+
+kind does not have any inbuilt way to provide IP addresses to your `Loadbalancer` type services, to ensure you get IP's you can install [MetalLB](https://kind.sigs.k8s.io/docs/user/loadbalancer/).
```

Comment:
```suggestion
Kind does not have any built-in way to provide IP addresses to your `Loadbalancer` service types. To ensure IP address assignments to Gateway Services, please consult [this guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for more information.
```

Reply:
Also need a blank line between this line and the header on line 74. This should fail linting

Reply:
Ah yes it did fail but couldn't figure out why, saw the output again and I see it now
```
./en/docs/setup/platform-setup/kind/index.md:73: MD022 Headers should be surrounded by blank lines
```